opensearch-project/OpenSearch-Dashboards
# [Workspace]Fix workspace assets page UI bugs

### Description

This PR is for fixing a bunch of UI bugs in the workspace assets page. Here are the fixed list:

- The spacing around the table and search bar should be 16px
- Remove the "Workspace" column and filter
- Add tooltip to "Refresh" icon button
- Change the label of the workspace select field to "Target workspace" and remove the field description
- Add "Close" button to the footer of the resulting message flyout
- Change "Objects" to "Assets" in the "Export" modal (all instances)
- Change "Saved objects" to "Assets" in the Import flyout
- Change "saved objects" to "assets" for the buttons



### Issues Resolved




## Screenshot


### Global assets
![image](https://github.com/user-attachments/assets/b5a39389-afc7-4864-842c-6283c1806cb1)


### Workspace assets
![image](https://github.com/user-attachments/assets/f3c1a950-ff4a-4d08-9b3d-7622198b2529)
![image](https://github.com/user-attachments/assets/91bf09fb-ad06-4d5a-afef-b5221eddad9a)
![image](https://github.com/user-attachments/assets/cb89fe2c-a2af-4a1f-abf3-1383b6f339db)
![image](https://github.com/user-attachments/assets/abb1cfe0-e3d9-4e22-a314-ff7e84356087)


## Testing the changes



- Clone branch code and run `yarn osd bootstrap --single-version ignore`
- Add below configs in `config/opensearch_dashboards.yml`
```
savedObjects.permission.enabled: true
workspace.enabled: true
uiSettings:
  overrides:
    'home:useNewHomePage': true
opensearchDashboards.dashboardAdmin.users: ['admin']
```
- Run `yarn start --no-base-path`
- Click "View all" in the "Assets" panel
- The global assets should work as screenshot above
- Enter a workspace use left navigation
- Click "View all" in the "Assets" panel
- The workspace assets page should work as screenshot above

## Changelog

- fix: [Workspace]Fix workspace assets page UI bugs

### Check List
